docker-compose-base.yml

```diff
@@ -33,3 +33,5 @@ server:
     - FXA_SECRET_KEY=a93644d71fa684da464fd2dba4d687c14cb21e1dd740af83895e70ede82f54cc
     - DATADOG_API_KEY=4f3b967bbf13c7769ac4fa89efda0fae
     - DATADOG_APP_KEY=ddd45a7b3a3cb90ff1baf20ae8cfab04d1937038
+    - ACCOUNT_INVITE_ONLY_MODE=True
+    - MOZILLIANS_API_KEY=ff33fa1255ce0ad59be17271093ab851c6357936
```

Comment:
I'm thinking they're okay, because we'll use different credentials & keys for production. They're just here for consistent dev environments that work out of the box without requiring new devs to dig up all these things on their own. Any data that ends up at these APIs is throwaway.